### ISSUE TITLE
[bug] stats double-prefixes step names in output

### ISSUE THREAD

**Issue by Luffy-nani** [rajudandigam/agent-inspect#172]
### Summary

The stats command displays step names with a doubled prefix instead of the correct single prefix that list and view show for the same run.

### Area

CLI (list/view/search/report/diff/check and friends)

### AgentInspect version

6.7.2

### Steps to reproduce

```bash
1. Wrap a run with `inspectRun`, using `step`, `step.tool`, and `step.llm`:

   await inspectRun("support-agent", async () => {
     await step("plan", async () => ({ intent: "refund-policy" }));
     const policy = await step.tool("retrieve-policy", async () => ({ text: "..." }));
     return step.llm("generate-answer", async () => `Policy: ${policy.text}`);
   }, { traceDir: "./.agent-inspect" });

2. Run `npx agent-inspect stats --dir ./.agent-inspect`
```

### Expected behavior

Step names shown as `tool:retrieve-policy`, `llm:generate-answer`

### Actual behavior

Step names are double-prefixed: `tool:tool:retrieve-policy`, `llm:llm:generate-answer`.

### Environment

Node v22.22.2, Linux (Ubuntu 24), npm

### Privacy confirmation

- [x] This report contains synthetic or redacted data only — no real prompts, production logs, tokens, or PII.
- [x] This is not a request for hosted observability, vendor upload, replay, or cost analytics (see docs/LIMITATIONS.md).

**Comment by abhay-codes07** [comment]
Root cause: `renderTraceStats` prints the "Slowest steps" rows as `${stepType}:${stepName}`, but `step.tool`/`step.llm` already store the name with its type prefix (`tool:retrieve-policy`, `llm:generate-answer`), so the prefix was applied twice. Plain `step()` names have no prefix, which is why `logic:plan` looked correct. `list`/`view` render the stored name as-is, so they were unaffected.

Fix adds the type prefix only when the name does not already start with it. Verified against the repro:

```
Slowest steps:
  run_... | tool:retrieve-policy | 28ms
  run_... | logic:plan | 14ms
  run_... | llm:generate-answer | 13ms
```

PR incoming.
